kafka-rest/src/main/java/io/confluent/kafkarest/controllers/TopicConfigManager.java

```diff
@@ -15,39 +15,37 @@
 
 package io.confluent.kafkarest.controllers;
 
-import io.confluent.kafkarest.entities.TopicConfiguration;
+import io.confluent.kafkarest.entities.TopicConfig;
 import java.util.List;
 import java.util.Optional;
 import java.util.concurrent.CompletableFuture;
 
 /**
- * A service to manage Kafka {@link TopicConfiguration TopicConfigurations}.
+ * A service to manage Kafka {@link TopicConfig TopicConfigs}.
  */
-public interface TopicConfigurationManager {
+public interface TopicConfigManager {
 
   /**
-   * Returns the list of Kafka {@link TopicConfiguration TopicConfigurations} belonging to the
+   * Returns the list of Kafka {@link TopicConfig TopicConfigs} belonging to the
    * {@link io.confluent.kafkarest.entities.Topic} with the given {@code topicName}.
    */
-  CompletableFuture<List<TopicConfiguration>> listTopicConfigurations(
-      String clusterId, String topicName);
+  CompletableFuture<List<TopicConfig>> listTopicConfigs(String clusterId, String topicName);
```

Comment:
although i think this is technically grammatically incorrect, it's harmless here and is nice and short.